### ISSUE TITLE
New edit view design and logic

### ISSUE THREAD

**Issue by IndrekV** [toggl/toggldesktop#1042]
Edit view is a slide out. Maybe this could be done so that edit view is in the same form and the form extends it's size when opening edit. This would remove the complicated popup position calculations.

- [x] Selected entry is highlighted in list, others have overlay on top of them
- [ ] Edit view slides out of main window
- [x] Edit view height is always the same as main window
- [x] Let's try to use the standard resize handle instead our custom one with the new design
- [x] Add task part should be left out of the first implementation
- [x] Description and project inputs are text inputs with autocomplete
- [x] Client is a combobox.
- [x] Items in project field show dot with project color and project name.
- [x] Pressing `Add project` or `Add Client` Switches the field with text field and shows `add` and `cancel` button.
- [x] Tags field is a text field that has autocomplete and labels inside it. User can start typing a new tag from the right side of the last tag label.
- [x] There is a slight shadow on the left side of the edit view.
- [x] Edit view has no border

### Edit View
![edit 1](https://cloud.githubusercontent.com/assets/842229/7908543/86efd4e0-084e-11e5-99fb-10e22b7248b9.png)

### Add Project
![add-project](https://cloud.githubusercontent.com/assets/842229/8107262/c51dd190-1050-11e5-8256-cce255e4d2ef.png)

### Remove tag
![remove-tag](https://cloud.githubusercontent.com/assets/842229/8107314/11b8d98c-1051-11e5-95fd-8f1cfb67b8ea.png)

**Comment by IndrekV** [comment]
#1285 should be done before this.

**Comment by IndrekV** [comment]
This has been started in branch `win-new-edit-design`

**Comment by amulware** [comment]
Original `win-new-edit-design` was merged into master while resolving #1349.
Now continuing on same branch after fast forward.

**Comment by IndrekV** [comment]
Changed the main description to add more details.

**Comment by HugoHeneault** [comment]
Wow. This is so awesome. I can't wait for it!!
